Rubberduck.RibbonDispatcher/AbstractRibbonDispatcher/RibbonButton.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.Office.Core;
+
+namespace RubberDuck.RibbonDispatcher {
+    using LanguageStrings     = IRibbonTextLanguageControl;
+
+    public class RibbonButton : RibbonCommon, IRibbonButton {
```

Comment:
Ah, and here comes the implementing types - I'd have them in some `Rubberduck.RibbonDispatcher.Concrete` or `.Concrete.Controls` namespace.

Reply:
Sounds good.